### ISSUE TITLE
good or bad practise together with angular-eslint

### ISSUE THREAD

**Issue by ProfEibe** [sindresorhus/eslint-plugin-unicorn#2055]
### Description

I haven't found any information about this online.
When setting up eslint in my angular project, there are already preconfigured 2 lint-plugins from angular.

Should I replace them with unicorn or just add the latter?

```
"extends": [
        "plugin:@angular-eslint/recommended",
        "plugin:@angular-eslint/template/process-inline-templates",
        "plugin:unicorn/recommended"
      ],
```

### Fail

```js

```

```js

```

### Pass

```js

```

```js

```

**Comment by sindresorhus** [comment]
This is not the place to ask general ESLint questions.
